examples/launch_counters.py throw an exception

<!--
For general questions, please ask on ROS answers: https://answers.ros.org, make sure to include at least the `ros2` tag and the rosdistro version you are running, e.g. `ardent`.
For general design discussions, please post on discourse: https://discourse.ros.org/c/ng-ros
Not sure if this is the right repository? Open an issue on https://github.com/ros2/ros2/issues
For Bug report or feature requests, please fill out the relevant category below
-->

## Bug report

**Required Info:**

- Operating System:
  - Ubuntu 16.04
- Installation type:
  - From source
- Version or commit hash:
  - a46aabdde1f58c7e2e5c056c0c4a1ea830faac5c
- DDS implementation:
  - Fast-RTPS, but shouldn't matter here
- Client library (if applicable):
  - N/A

#### Steps to reproduce issue
```
$ cd <path-to-launch>/launch/examples
$  python3 ./launch_counters.py 
```

#### Expected behavior
Script runs successfully
#### Actual behavior
```sh
[ERROR] [launch]: exception occurred while executing process[whoami-1]:
Traceback (most recent call last):
  File "./../launch/actions/execute_process.py", line 423, in __execute_process
    stderr_to_stdout=(self.__output == 'screen'),
  File "<path-to-workspace>/install/lib/python3.5/site-packages/osrf_pycommon/process_utils/async_execute_process_asyncio/impl.py", line 140, in async_execute_process
    stderr_to_stdout)
  File "<path-to-workspace>/install/lib/python3.5/site-packages/osrf_pycommon/process_utils/async_execute_process_asyncio/impl.py", line 48, in _async_execute_process_nopty
    stderr=stderr, close_fds=False)
  File "/usr/lib/python3.5/asyncio/base_events.py", line 1079, in subprocess_exec
    bufsize, **kwargs)
  File "/usr/lib/python3.5/asyncio/unix_events.py", line 184, in _make_subprocess_transport
    **kwargs)
  File "/usr/lib/python3.5/asyncio/base_subprocess.py", line 40, in __init__
    stderr=stderr, bufsize=bufsize, **kwargs)
  File "/usr/lib/python3.5/asyncio/unix_events.py", line 649, in _start
    universal_newlines=False, bufsize=bufsize, **kwargs)
  File "/usr/lib/python3.5/subprocess.py", line 947, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.5/subprocess.py", line 1551, in _execute_child
    raise child_exception_type(errno_num, err_msg)
FileNotFoundError: [Errno 2] No such file or directory: 'time'
```
#### Additional information

--- a/launch/examples/launch_counters.py
+++ b/launch/examples/launch_counters.py
@@ -71,7 +71,7 @@ def main(argv=sys.argv[1:]):
         whoami_cmd = [launch.substitutions.FindExecutable(name='whoami')]
     whoami_action = launch.actions.ExecuteProcess(
         cmd=whoami_cmd,
-        shell=(platform.system() == 'Windows')
+        shell=True
     )
     ld.add_action(whoami_action)
     # Make event handler that uses the output.

